nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -299,40 +324,29 @@ def _get_bias_shift(
         raw_output = engine.infer(input_blob)
         q_outputs = self._backend_entity.process_model_output(raw_output, output_name)
         q_outputs = self._backend_entity.tensor_processor.mean_per_channel(q_outputs, channel_axis).tensor
-        bias_shift = np.array(output_fp) - q_outputs
+        bias_shift = self._backend_entity.post_process_output_data(output_fp) - q_outputs
         return bias_shift
 
-    @staticmethod
-    def _get_bias_shift_magnitude(current_bias_value: np.ndarray, updated_bias_value: np.ndarray) -> float:
-        """
-        Calculates bias shift magnitude based on the current and updated values.
-
-        :param current_bias_value: The original bias value.
-        :param updated_bias_value: The updated bias value.
-        :return: Magnitude between original and updated bias values.
-        """
-        bias_shift_magnitude = np.inf
```

Comment:
Could you implement this logic using `NNCFTensor`? cc' @daniil-lyakhov

Reply:
What do you mean?

Reply:
We have abstractions `NNCFTensorProcessors` and `NNCFTensor` to implement shared logic of work with tensor. Is it possible to use it here?